docs/usage.md

```diff
@@ -393,3 +393,33 @@ We recommend adding the following line to your environment to limit this (typica
 ```bash
 NXF_OPTS='-Xms1g -Xmx4g'
 ```
+
+### Running the pipeline without Internet access
+
+The pipeline and container images can be downloaded using [nf-core tools](https://nf-co.re/docs/usage/offline).
+Here is an example command to download pipeline version 1.1.0 with singularity images:
+
+```
+nf-core download \
+    --container singularity \
+    --compress none \
+    --revision 1.1.0 \
+    raredisease
+```
+For running offline, you of course have to make all the reference data available locally, and specify `--fasta`, etc., see [above](#reference-files-and-parameters).
+
+Contrary to the paragraph about Nextflow on the nf-core tools page, it is not possible to use the "-all" packaged version of Nextflow. The online version of nextflow is necessary to support the nextflow plugins used for this pipeline. Download instead the file called just `nextflow`. Nextflow will download its dependencies when it is run. Additionally, you need to download the nf-validation plugin explicitly:
```

Comment:
Not sure which homepage and packages you are referring to there. Perhaps you mean https://nf-co.re/tools  ?

I didn't find any "paragraph about Nextflow" on https://nf-co.re/tools

Reply:
I've added another link, hope it makes sense like that